core/admin/EE_Admin_Page.core.php

```diff
@@ -1402,6 +1402,14 @@ protected function _add_help_tour()
                 $this->_help_tour[ $route ][] = EEH_Template::help_tour_stops_generator($end_stop_tour);
             }
         }
+
+        /**
+         * Allow extending the help tours variable.
+         *
+         * @param Array $tours The array containing all tour items to be displayed.
+         */
+        $tours = apply_filters('FHEE__EE_Admin_Page___add_help_tour__tours', $tours);
```

Comment:
i think this filter will need to be applied to the ` $this->_help_tour` array after the conditional below.
You will notice in the code above that some things like:

```php
$this->_help_tour[ $route ][] = EEH_Template::help_tour_stops_generator($end_stop_tour);
```

are getting added directly to ` $this->_help_tour` and therefore would not be editable via the filter you have added.